index.html

```diff
@@ -546,7 +548,7 @@
                 </tbody>
             </table>
             <p>In the case of the <a>primary pointer</a>, these events (with the exception of <code>gotpointercapture</code>, and <code>lostpointercapture</code>) may also fire <a>compatibility mouse events</a>.</p>
-            <p>For all PointerEvents in the table above, <a href="https://www.w3.org/TR/uievents/#widl-UIEvent-detail">detail</a> attribute defined in [[!DOM-LEVEL-3-EVENTS]] SHOULD be 0.</p>
+            <p>For all PointerEvents in the table above, <a href="https://www.w3.org/TR/uievents/#widl-UIEvent-detail"><code>detail</code></a> attribute defined in [[!DOM-LEVEL-3-EVENTS]] SHOULD be 0. For <code>gotpointercapture</code> and <code>lostpointercapture</code> all the attributes except the ones defined in the table above should be the same as the Pointer Event that causes the user agent to run <a href="#process-pending-pointer-capture">Process Pending Pointer Capture</a> and fire <code>gotpointercapture</code> and <code>lostpointercapture</code> events.</p>
```

Comment:
I realize now that this bit about `detail` really fits better in the "firing events" section (since that's what's describing how events get fired, including setting other properties like `bubbles` and `cancelable`.  Mind moving it?

Then I think copying the properties for `got/lostpointercapture` make the most sense in the "process pending pointer capture" section.  In fact I think the property copying also applies to all the other events fired there (over,enter,leave,out) too, right?  So maybe just one sentence at the top of 5.1.3.1 indicating that this is done only in response to some other pointer event, and (unless indicated otherwise) all events fired in that section get their property values from the original event.  WDYT?